library/build.gradle

```diff
@@ -28,8 +28,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
lambdas will not be backported. (this is part of the gradle android plugin 3.+). Better leave it at 1.7. We don't want retrolambda as dependency